### ISSUE TITLE
Allow unauthenticated browsers to view games

### ISSUE THREAD

**Issue by irilyth** [buttonmen-dev/buttonmen#1483]
It'd be nice if browsers could view games without being logged in, so we could send links to things like story-tournament games to people who don't play Button Men at all.

**Comment by irilyth** [comment]
I mentioned this on #625, and meant to create a ticket for it, but never got around to it.

I don't have a strong opinion about whether this should only work for games where chat is public, or all games, or all games not marked as private in some other way, or what. "All games where chat is public" seems like a reasonable and simple starting point, unless anyone has ideas about situations where we'd want something more complicated.

**Comment by yawetag** [comment]
Is there a spot between "public" and "private" that's similar to "logged in
users"? If so, I'm fine with the game being viewed at the same level as the
chat.


Andrew Senger
asenger@gmail.com

**Comment by irilyth** [comment]
Right now, chat privacy is only a glimmer in the eye of #464, so on the one hand nothing is implemented, but on the other hand anything is possible. :^)

IMHO it'd be fine to have chat privacy settings for "logged-in users", "private", and "public", and to allow unauthenticated users to only see games where the chat is public. But #464 is probably the place to hash that out.

Really I just wanted to make this ticket as a placecholder, so that I could delete from my inbox the message reminding me to create the ticket. :^) I think it's reasonable for it to depend on #464.

**Comment by yawetag** [comment]
Agreed, and was only answering your question. I think I may have given my
opinion already on #464. If not, I'll do so.


Andrew Senger
asenger@gmail.com
